README.md

```diff
@@ -1 +1,156 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> This is my portfolio I build it with html and css.
+
+**[Hello Microverse]** is the first project I officially created on day 2 of my first week studying software programming.
+
+## 🛠 Built With <a name="built-with"></a>
+
+1. html
+2. css
+3. Linters
+
+### Tech Stack <a name="tech-stack"></a>
+
+1. html
+2. css
+3. Linters
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://indexhtml.org/">HTML</a></li>
+    <li><a href="https://stylecss/">CSS</a></li>
+    <li><a href="https://linters.org/">Linters</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+index.html file.
+style.css file.
+Node.js modules
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Web browser
+Code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+Run this command:
+git clone https://github.com/Mike47ip/Hello_Microverse_Project.git
+
+### Install
+
+Install this project with:
+
+cd my-project
+npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using live server extension.
+
+### Run tests
+
+To run tests, run the following command:
+
+Coming soon.
+
+### Deployment
+
+Coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="Mike"></a>
+
+👤 **Mike**
+
+- GitHub: [@githubhandle](https://github.com/Mike47ip)
+- Twitter: [@twitterhandle](https://twitter.com/Mikepee47)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/MichaelOseiDarkwah)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add header
+- [ ] Add body
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
```

Comment:
:red_circle: Please consider completing this sentence to make this readme looks professional. You could say something like this :arrow_down: 
> If you like this project, please consider giving it a star on GitHub. This will help other people find and use the project. You can also contribute to the project by submitting bug reports, feature requests, or pull requests.